staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1277,3 +1283,51 @@ type RenameOptions struct {
 	NewName         string  `json:"newName"`
 	OldName         *string `json:"oldName,omitempty"`
 }
+
+// KubeVirtConfiguration holds all kubevirt configurations
+// +k8s:openapi-gen=true
+type KubeVirtConfiguration struct {
```

Comment:
This could be renamed to VirtualizatioConfiguration

KubeVirtConfiguration leaves more the impression of "system wide settings", but many settings in this group are really virt specific.

But this is nit picking on a high level